src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -22,7 +22,7 @@ internal partial class SNIProxy
     {
         private const int DefaultSqlServerPort = 1433;
         private const int DefaultSqlServerDacPort = 1434;
-        private const string SqlServerSpnHeader = "MSSQLSvc";
+        private const string SqlServerSpnHeader = "MSSQLSvc/";
```

Comment:
should we use '@' here to avoid need to copy & replace later?

Reply:
The "@" format is only for Linux.  The Windows format always use "/". The current code (before this PR) has different files for Windows vs. Linux. This PR undoes that to go back to what it was before I made those change a month ago.

The problem is that this managed code here in CoreFx for SqlClient builds as separate package.  And that package can run against an older .NET Core 2.1 runtime.  And that runtime which uses System.Net.Security.Native doesn't understand the "@" format.  So, right now the latest pre-release packages of System.Data.SqlClient (based on this master branch, .NET Core 3.0) is broken. So, this PR is trying to keep the managed layer the same as what it was in .NET Core 2.1.  

This is why the logic for transformation of "/" to "@" moves to the C layer which doesn't build as a separate (out-of-band) package.